digitalpublicgoods/openmrs.json

```diff
@@ -0,0 +1,101 @@
+    },
+    "inappropriateIllegalContent": {
+      "collectStoreDistribute": "Yes",
+      "type": "Documentation, contributor comments",
+      "contentFilter": "No",
+      "policyGuidelinesDocumentationLink": "",
+      "illegalContentDetection": "Yes",
+      "illegalContentDetectionMechanism": "OpenMRS prides itself on being a friendly and collaborative community. This comes largely from our ability to hold ourselves, and each other, accountable for our behavior with each other. Part of that approach to accountability includes escalating awareness of circumstances that make members of our community feel unequal, uncomfortable, or unsafe. In situations where someone witnesses behaviors in the community that betrays our code of conduct, we encourage individuals to communicate concerns directly with that individual or group. In circumstances where someone is either uncomfortable approaching the individual or group, or feel as if their intervention has been unsuccessful, individuals are encouraged to approach the OpenMRS community management team."
+    },
+    "protectionFromHarassment": {
+      "userInteraction": "Yes",
+      "addressSafetySecurityUnderageUsers": "No",
+      "stepsAddressRiskPreventSafetyUnderageUsers": [
+        "We do not identify the target age of our contributors, enforce age limits, or provide any parental/guardian controls."
+      ],
+      "griefAbuseHarassmentProtection": "Yes",
+      "harassmentProtectionSteps": [
+        "Please see the OpenMRS Code of Conduct (https://wiki.openmrs.org/display/RES/OpenMRS+Code+of+Conduct). Note that the OpenMRS community management team considers and decides upon an appropriate penalty based on the severity of the violation. In all instances, both the reporter and violator will be notified of the event, and what action will be taken. Such actions could include one or more of the following:",
+        "An official warning message on the violation",
+        "Temporary or permanent suspension of OpenMRS accounts",
+        "Removal of privileges from, or adding restrictions to OpenMRS accounts",
+        "Removal of content/contributions",
+        "Removal from the OpenMRS community"
+      ]
+    }
+  },
+  "locations": {
+    "developmentCountries": [
+      ""
+    ],
+    "deploymentCountries": [
```

Comment:
Though not required, it would be really nice to have countries listed here. Thoughts? @nathanbaleeta

Reply:
Resolved in commit a51f6d5